lib/site_web/controllers/static_page_controller.ex

```diff
@@ -0,0 +1,15 @@
+defmodule SiteWeb.StaticPageController do
+  use SiteWeb, :controller
+
+  def index(conn, _params) do
+    render(conn, "index.html", year: 2018)
+  end
+
+  def la(conn, _params) do
+    render(conn, "la_retreat_2018.html", year: 2018, title: "EMPEX LA Civic Hack Retreat 2018")
+  end
+
+  def show(conn, %{"city" => city, "type" => type, "year" => year}) do
+    render(conn, "#{city}_#{type}_#{year}.html", year: 2018)
```

Comment:
how does `type` work for the regular conference?  `la_conference_2019`?

Reply:
and is this action to support historical events?

Reply:
yeah - right now I've got it so that `/la/conf/2018` will link to `la_conf_2018.html.eex`, as a way to handle historical events.